app/models/content_item_subscriber_list.rb

```diff
@@ -32,20 +32,38 @@ def subscription_params
   def link_hash
     case content_item_type
     when 'taxon'
-      taxon_links
+      taxon_or_world_location_links
     when 'organisation'
       organisation_links
+    when 'person'
+      person_links
+    when 'ministerial_role'
+      ministerial_role_links
+    when 'topical_event'
+      topical_event_links
+    when 'world_location'
+      world_location_links
     else
       message = "No link hash available for content items of type #{content_item_type}!"
       raise UnsupportedContentItemError, message
     end
   end
 
-  def taxon_links
+  def taxon_or_world_location_links
+    if content_item['base_path'].match(%r{^/world/(.*)})
+      world_location_links
```

Comment:
I'd probably just put the world location links hash here since it's very short. Otherwise this seems to have an in-balance with the else condition.

Reply:
Sure